docs/rules/no-standalone-expect.md

```diff
@@ -0,0 +1,60 @@
+# No standalone expect in a describe block (no-standalone-expect)
+
+Prevents expects outside of a test or it block. An expect within a helper
+function (but outside of a test or it block) will not trigger this rule.
+
+## Rule Details
+
+This rule aims to eliminate expects that will not be executed. An expect inside
+of a describe block but outside of a test or it block or outside of a describe
+will not execute and therefore will trigger this rule. It is viable, however, to
+have an expect in a helper function that is called from within a test or it
+block so expects in a function will not trigger this rule.
+
+Examples of **incorrect** code for this rule:
+
+```js
+// in describe
+describe('a test', () => {
+  expect(1).toBe(1);
+});
+
+// below other tests
+describe('a test', () => {
+  it('an it', () => {
+    expect(1).toBe(1);
+  });
+
+  expect(1).toBe(1);
+});
+```
+
+Examples of **correct** code for this rule:
+
+```js
+// in it block
+describe('a test', () => {
+  it('an it', () => {
+    expect(1).toBe(1);
+  });
+});
+
+// in helper function
+describe('a test', () => {
+  const helper = () => {
+    expect(1).toBe(1);
+  };
+
+  it('an it', () => {
+    helper();
+  });
+});
+```
+
+\*Note that this rule will not trigger if the helper function is never used even
```

Comment:
why the `\*`?

Reply:
Oh I didn't mean the \ just meant a * there

Reply:
Still here. I guess prettier adds it? `*` is used to make things bold, and when you only have one, prettier makes sure to escape it